dspace-api/src/main/java/org/dspace/disseminate/CitationDocumentServiceImpl.java

```diff
@@ -272,7 +221,7 @@ public Boolean isCitationEnabledForBitstream(Bitstream bitstream, Context contex
         }
 
         // If previous logic didn't return true, then we're false.
-        return false;
```

Comment:
Why do you change the default return value (cover page enhanced PDF is not generated)?

Reply:
good catch, i must have changed while trying things out. was not intended to be part of the change